package.json

```diff
@@ -3,6 +3,7 @@
   "version": "6.2.8",
   "description": "FantasyLand compliant (monadic) alternative to Promises",
   "main": "index.js",
+  "typings": "index.d.ts",
```

Comment:
It seems that `"typings"` is an alias for `"types"`. Let's use `"types"` instead.